packages/intrepid2/src/Shared/Intrepid2_Types.hpp

```diff
@@ -122,11 +122,11 @@ namespace Intrepid2 {
     /// The maximum reconstruction order.
     static constexpr ordinal_type MaxOrder             = 8;
     /// The maximum number of integration points for direct cubature rules.
-    static constexpr ordinal_type MaxIntegrationPoints = 1001;    
+    static constexpr ordinal_type MaxIntegrationPoints = 4893;    
     /// The maximum degree of the polynomial that can be integrated exactly by a direct edge rule.
     static constexpr ordinal_type MaxCubatureDegreeEdge= 20;      
```

Comment:
Agreed. 
It shouldn't be hard to compute the  CubatureDirectLineGaussJacobi20 for higher order.
And I agree it would be nice to provide on-the-fly computation of Gauss Jacobi points and go to arbitrary order.